public/components/datasources/components/__tests__/__snapshots__/datasource.test.tsx.snap

```diff
@@ -0,0 +1,309 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Not sure how this got pulled in, should be on main already, can you double check/merge